proteus/Gauges.py

```diff
@@ -310,13 +311,17 @@ def initOutputWriter(self):
 
         numLocalQuantities = sum([len(self.measuredQuantities[field]) for field in self.fields])
         self.localQuantitiesBuf = np.zeros(numLocalQuantities)
-
         if self.gaugeComm.rank != 0:
             self.globalQuantitiesBuf = None
             self.globalQuantitiesCounts = None
         else:
-            self.fileName = os.path.join(Profiling.logDir, self.fileName)
-            self.file = open(self.fileName, 'w')
+            if self.adapted:
```

Comment:
Good point. It wasn't throwing an error but I'll need to add a close file statement somewhere before the resetting of the gauges.